### ISSUE TITLE
Mod Organizer not updating .txt files correctly and .ini file problem.

### ISSUE THREAD

**Issue by doc97** [TanninOne/modorganizer#623]
Win 10 64bit
Alpha 4 (v2.0.4)
Admin account, running MO as Admin.
No files set to Read-Only

.TXT FILE PROBLEM, EDITING NOT WORKING CORRECTLY:
Add mod into MO through Nexus>Enable it>Load Game>Mod enabled>Exit Game>Disable mod in MO>Load Game>Mod Still enabled>Check Modlist.txt in profiles folder>Mod is still listed in the file!

.INI FILE PROBLEM
Mod Organizer is NOT using the fallout4.ini and fallout4prefs.ini in Mod Organizers profiles/profile name folder, but uses the .ini files in Documents/My games/Fallout4
Could have seen this mentioned in another Issue before, not sure.

Sorry if I missed anything/did something wrong or was unclear, haven't posted bug reports before!
- Reliikki

**Comment by rdunlap** [comment]
These are both known issues in Alpha 4. For issue 1 if you click the dropdown and select refresh it will refresh its data. As for the Ini files from the profile this appeared in Alpha 4 and i would expect it to be resolved with the next revision. You can just copy them into the default locations until then. 

Robert

**Comment by Nik-Lz** [comment]
Indeed. Mod Organizer uses the ini files in Documents\My Games\Fallout4\ only if the profile you set doesn't have "Local Game Settings" checked. Careful.
